src/polkadot/PolkadotEngine.ts

```diff
@@ -637,3 +890,62 @@ export async function makeCurrencyEngine(
 
   return engine
 }
+
+export interface LiberlandTxProcessingContext {
+  walletId: string
+  walletInfo: SafePolkadotWalletInfo
+  currencyInfo: EdgeCurrencyInfo
+  allTokensMap: EdgeTokenMap
+  tokenId: EdgeTokenId
+}
+
+export function processLiberlandTransaction(
+  context: LiberlandTxProcessingContext,
+  tx: LiberlandTransfer
+): EdgeTransaction | undefined {
+  const { id, fromId, value, block } = tx
+  const { tokenId } = context
+
+  const blockHeight = parseInt(block.number)
+  const date = new Date(block.timestamp).getTime() / 1000 // Convert timestamp to epoch seconds
```

Comment:
I looked at the return object and the timestamp appears to be UTC but doesn't include the Z at the end. You can add it to the end to ensure it's handled correctly